human-mutation.csl

```diff
@@ -13,11 +13,11 @@
     <category field="medicine"/>
     <issn>1059-7794</issn>
     <eissn>1098-1004</eissn>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2016-12-27T20:07:23+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author-short">
-    <names variable="author">
+    <names variable="author" suffix=",">
```

Comment:
@damnation333, while this works, styles are easier to understand and edit if you put separating punctuation at the highest possible level. In particular, try to write macros in such a way that they don't generate output with dangling separating punctuation (i.e. "Miller et al." is better than "Miller et al.,").

In this case, it's best to change:

```xml
    <layout prefix="(" suffix=")" delimiter="; ">
      <text macro="author-short"/>
      <text macro="issued" prefix=" "/>
    </layout>
```

to

```xml
    <layout prefix="(" suffix=")" delimiter="; ">
      <group delimiter=", ">
        <text macro="author-short"/>
        <text macro="issued"/>
      </group>
    </layout>
```